conformance/reports/v1.3.0/cloud-provider-kind/standard-v0.8.0-alpha.1-default-report.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: "2025-09-07T12:44:41Z"
+gatewayAPIChannel: standard
+gatewayAPIVersion: v1.3.0
+implementation:
+  contact:
+  - antonio.ojea.garcia@gmail.com
```

Comment:
Lots also just link to GH issues page if you don't want to put your email, ie https://github.com/kubernetes-sigs/gateway-api/blob/368d492fffe6dc503312003f65c228e294f7947e/conformance/reports/v1.3.0/nginx-nginx-gateway-fabric/experimental-2.0.0-default-report.yaml#L7

Reply:
updated